vmm/src/lib.rs

```diff
@@ -1673,10 +1684,10 @@ impl Vmm {
         LOGGER.set_include_origin(api_logger.show_log_origin, api_logger.show_log_origin);
         LOGGER.set_include_level(api_logger.show_level);
 
-        let options = match api_logger.options {
-            Value::Array(options) => options,
-            _ => vec![],
-        };
+        #[cfg(target_arch = "aarch64")]
+        let options: &Vec<Value> = &vec![];
```

Comment:
This will prevent any future log options from being configured on ARM. I think it would be cleaner to remove the `LogDirtyPages` enum variant for `aarch64` and fail at deserialization. Something like:
```diff
iff --git a/logger/src/lib.rs b/logger/src/lib.rs
index b2059b9..fe8ddbe 100644
--- a/logger/src/lib.rs
+++ b/logger/src/lib.rs
@@ -209,6 +209,7 @@ enum Destination {
 #[repr(usize)]
 pub enum LogOption {
     /// Enable KVM dirty page tracking and a metric that counts dirty pages.
+    #[cfg(target_arch = "x86_64")]
     LogDirtyPages = 1,
 }
 ```

Reply:
I don't think you can have an empty enum. It will generate an error. But I can try this.

Reply:
`LogOption` has its variants as usize. We can't have an empty enum in this case:
```
error[E0084]: unsupported representation for zero-variant enum                                                                                                                                                                                                                     
   --> logger/src/lib.rs:215:1                                                                                                                                                                                                                                                     
    |                                                                                                                                                                                                                                                                              
215 |   #[repr(usize)]                                                                                                                                                                                                                                                             
    |   ^^^^^^^^^^^^^^                                                                                                                                                                                                                                                             
216 | / pub enum LogOption {                                                                                                                                                                                                                                                       
217 | |     /// Enable KVM dirty page tracking and a metric that counts dirty pages.                                                                                                                                                                                               
218 | |     #[cfg(target_arch = "aarch64")]                                                                                                                                                                                                                                        
219 | |     LogDirtyPages = 1,                                                                                                                                                                                                                                                     
220 | | }                                                                                                                                                                                                                                                                          
    | |_- zero-variant enum 
``` 
I say we leave it as is for the moment and in case we want to add different options that are also valid on arm, we can make this change then.